src/components/organisms/AnimationPage.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
```

Comment:
I think the pages might be templates, not organisms, but I'm not sure. @ChuckBTaylor might feel different.

Reply:
Definitely a template (if not a page)

Reply:
Great, I'll move all of these files I created from the Organisms into the Templates.

Reply:
Very strangely - when the `Home.tsx` file is inside of Organisms in my VSCode the code is all colorful and pretty:
<img width="839" alt="Screen Shot 2024-01-08 at 8 42 27 AM" src="https://github.com/mwkwsd/sensensomething/assets/102488171/1e6ded8e-4d6e-4471-bc99-9087d6fc025d">

Move it into the Templates folder and it goes all white. Still works totally fine just a weird VSCode thing:
<img width="760" alt="Screen Shot 2024-01-08 at 8 43 55 AM" src="https://github.com/mwkwsd/sensensomething/assets/102488171/20d6dade-feda-4e58-9a28-67e89ededbbb">